editor/src/DocumentWindows/MaterialInspector.cpp

```
@@ -0,0 +1,102 @@
+//
+///////////////////////////////////////////////////////////////////////////////
+
+#include "MaterialInspector.hpp"
+#include "DocumentWindows/InspectorWindow.hpp"
+#include "utils/ScenePreview.hpp"
+#include "components/Camera.hpp"
+#include "Components/Widgets.hpp"
+#include "context/Selector.hpp"
+
+namespace nexo::editor {
+	void MaterialInspector::setup()
```

Cppcheck (2.10-2)

[style] The function 'setup' is never used.

(unusedFunction)